packages/react-generator/src/generator/page.js

```
@@ -0,0 +1,761 @@
+    const subTemplate = children.map((child) => {
+      // 如果子元素是 JSExpression 或 JSDataBinding，直接处理
+      if (child?.type === 'JSExpression' || child?.type === 'JSDataBinding') {
+        const convertedValue = cleanStateReference(child.value, isClassComponent)
+        return `{ ${convertedValue} }`
+      }
+      // 如果子元素是纯文本，直接返回
+      if (typeof child === 'string' && child.trim()) {
+        return child
+      }
+      // 否则递归处理
+      return generateJSXNode(child, state, description, false, isClassComponent)
```

ESLint

[error] 'generateJSXNode' was used before it was defined.

(@typescript-eslint/no-use-before-define)